src/tests/test_kube_api.py

```diff
@@ -18,49 +20,71 @@
     AgentClusterInstall,
     ClusterDeployment,
     InfraEnv,
+    KubeAPIContext,
     Proxy,
     Secret,
 )
 from service_client import log
 from tests.base_kubeapi_test import BaseKubeAPI
 from tests.config import ClusterConfig, InfraEnvConfig, global_variables
+from tests.global_variables import get_default_triggers
 
 PROXY_PORT = 3129
 
 
 class TestKubeAPI(BaseKubeAPI):
-    @JunitTestSuite()
-    @pytest.mark.kube_api
-    def test_kube_api_ipv4(self, kube_test_configs_single_node, kube_api_context, get_nodes):
-        namespace = global_variables.spoke_namespace
-        cluster_config, tf_config = kube_test_configs_single_node
-        self.kube_api_test(kube_api_context.api_client, get_nodes(tf_config, cluster_config), cluster_config, namespace)
+    KUBEAPI_IP_OPTIONS = [(False, True), (True, False)]
```

Comment:
I might prefer to embed this in the parametrize call
because otherwise it loses its meaning having it just boolean values